arrow-core/src/main/kotlin/arrow/core/extensions/const/traverseFilter/ConstTraverseFilter.kt

```diff
@@ -30,22 +30,14 @@ internal val traverseFilter_singleton: ConstTraverseFilter<Any?> = object :
   "EXTENSION_SHADOWED_BY_MEMBER",
   "UNUSED_PARAMETER"
 )
-@Deprecated(
-  "@extension kinded projected functions are deprecated",
-  ReplaceWith(
-  "traverseFilter(arg1, arg2)",
-  "arrow.core.traverseFilter"
-  ),
-  DeprecationLevel.WARNING
-)
+@Deprecated("Kind/type constructors will be deprecated, so this typeclass will no longer be available from 0.13.0")
```

Comment:
Same here for traverseFilter, filterMap, filterA, filter?

Reply:
What do u mean by same here? those are deprecated too, right? or you mean something else?

Reply:
Oh, seems my internet broke in the middle of the review. I was asking in another file, wether those functions have a replacement in their concrete DT? Or those are Deprecated without an direct replacement?

Reply:
ahh okay! I don't know actually.. maybe @nomisRev can bring some light to that question?

Reply:
They don't need to be replaced in `Const` IMO. There is not much value for these in `Const`. Less is more for 1.0.0, it's easier to add new methods in the future than to remove them.